package.json

```diff
@@ -24,7 +24,7 @@
   "license": "ISC",
   "devDependencies": {
     "@types/jest": "^24.0.1",
-    "bitmovin-player": "^8.6.0",
+    "bitmovin-player": "8.6.0",
```

Comment:
Doesn't it make more sense to keep this as `^`

Reply:
I thought about it too but we had it in the past that types were changing and therefore we needed to adapt code. Especially in the unit tests.